nestauk/dsp_interview_transcripts
# Add BIT France analysis

To test this PR:

* follow the setup steps in `dsp_interview_transcripts/pipeline/bit_france/README.md`
* run `dsp_interview_transcripts/pipeline/bit_france/analysis/run_bit_france_analysis.py`
* navigate to `dsp_interview_transcripts/pipeline/bit_france/report/bit_france_report.qmd` and press `SHIFT`+`CTRL`+`k` to render the report

This should produce an html report with some nice plots!

Areas to consider for feedback:

- Is there scope for further refactoring of `dsp_interview_transcripts/utils/topic_modelling.py` and `dsp_interview_transcripts/utils/llama_utils.py`? Should there be just one class that does everything (topic modelling and summarisation)?
- Is it clear what's going on - are the current READMEs and docstrings sufficient?

... plus anything else that you spot!
